bindings/kafka/0.4.0/message.json

```diff
@@ -11,12 +11,15 @@
   },
   "properties": {
     "key": {
-      "oneOf": [
+      "anyOf": [
         {
           "$ref": "http://asyncapi.com/definitions/3.0.0/Reference.json"
         },
         {
           "$ref": "http://asyncapi.com/definitions/3.0.0/schema.json"
+        },
+        {
+          "$ref": "http://asyncapi.com/definitions/3.0.0/avroSchema_v1.json"
         }
```

Comment:
maybe the protobuff schema definition must be included too ? 

And why oneOf does not work with this additional value ? i'm not sure to understand

Reply:
> maybe the protobuff schema definition must be included too ?

we cannot add Protobuf because it is not in kafka binding spec -> https://github.com/asyncapi/bindings/tree/master/kafka#message-binding-object

> And why oneOf does not work with this additional value ? i'm not sure to understand

because `schema.json` also defines `$ref` property. And `oneOf` means that schema should be valid ONLY against one of listed schemas. So if Reference Object schema and `schema.json` both talk about `$ref` - there is a conflict - at least this is my JSON Schema understanding

Reply:
ha yes ok !